### PR TITLE
onboarding pages and post cards styling

### LINKED ISSUE #25: Styling: Onboarding Pages & Post Cards
## 📝 Task Description 

Style the Welcome, Login/Sign-Up pages, and post cards according to the [Figma](https://www.figma.com/file/pxgrEsnOJneqsaWWeEnQoY/FreeBites?type=design&node-id=2904%3A12611&mode=design&t=ASnzrFFtsl0cB5Ux-1).

## ✅ Acceptance Criteria 

- [x] Is the welcome page styled as per the Figma?
- [x] Is the login page styled as per the Figma?
- [x] Is the sign-up page styled as per the Figma?
- [x] Are the post cards styled as per the Figma?
- [x] Are the modals (pop-ups) made as per the Figma?

## 🔍  Subtask Breakdown

- Style the welcome page
  - FreeBites logo and text
  - Login/signup buttons & touch effects
  - Overall spacing & sizing changes
- Style the login/signup pages
   - Logo, page title
   - Email / Password fields
   - Checkbox & text
   - Get Started/Login Button
   - Don’t have an account? Sign Up or Have an account? Login 
- Posts/Card changes (component --> HomePost.tsx)
   - Overall changes to the look/style
    - ‼️ Posts made by the user should have a delete button → when pressed it leads to a “Are you sure you want to delete?” modal (style this too please!)

** As you work, ensure to implement any touch (hover) effects for clickable objects **

## 💫  Recommended Approach

A suggestion is to start with the Welcome page → Login/Signup → Post Cards → Modals, as that's in increasing level of difficulty. Since this ticket also consolidates a bunch of pages – feel free to make separate PRs as you finish each page~

Pulling up the Figma and utilizing Dev Mode & Preview Mode helps get a good sense of what the app should look like for the user, aiding you guys as you code! This is particularly important for seeing the various effects buttons should have when a user clicks it, or to sanity check that your implementation matches the Figma.

Moreover, you’ll notice that some of the code will be repetitive so reuse it (++ points for abstraction yay!!) 

## ✨ Styling Resources ✨

- [How to CSS](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/CSS_basics)
- [Clarence's Styling Presentation](https://docs.google.com/presentation/d/10k0kGCUdR3twBIv9yDJ0hOQZCDFyoy99SGMMBguShFg/edit#slide=id.g24e07eed7b0_1_75)

### PR DESCRIPTION
**Pull request by vle04**
Styled landing, login, and signup pages to be accurate to the figma, as well as the post cards on the home page.

# Description

Changed the styling for the initial landing page, login page, and sign up page. The login and sign up are now split up into two separate pages. Also styled the post cards in the home page feed to be accurate to the figma. The delete button was implemented in a way where the overall look of the post card wouldn't change if it was removed (for all posts and your posts).

## Issues

Resolve #25 

## Screenshots

IMG_0976
IMG_0977
IMG_0978
IMG_0979

## Test

By running the app on expo to see the physical changes!

## Possible Downsides

The biggest issue we had with this ticket was trying to change the pngs to svgs, which we could not figure out. Additionally, we did not know how to change the title in the landing page to just text with a font (we don't know how to use fonts). We also did not have time to make it so that each post card has their own respective information (we just used placeholders).

### CODE REVIEW

File: src/app/(auth)/login.tsx [hunk 1]
```diff
@@ -11,17 +11,22 @@ export default function SignIn() {
     <SafeAreaView style={[globalStyles.container, { alignItems: "center" }]}>
       <Welcome />
 
-      <View style={{ gap: 23, flex: 1, width: "100%", alignItems: "center" }}>
-        <Link href={{ pathname: "/signup", params: { login: "true" } }} asChild>
-          <LoginButton text="login" />
+      <View style={{ gap: 23, flex: 1, width: "100%", alignItems: "center", marginTop: 86, }}>
```
Comment (LordofMankid) [inline src/app/(auth)/login.tsx:14]: not your code, bbut can you just put this in StyleSheet?

**Review by LordofMankid — COMMENTED** [review]
great job guys! it looks really really good. try doing a onceover to remove any commented out code that you don't think is needed anymore (don't be scared we'll see what you delete so we'll just say not to delete it)

feel free to get started on the changes, and read up on [react-native-svg](https://docs.expo.dev/versions/latest/sdk/svg/), as I'm asking sarah to provide you guys with SVG files for tag icons cuz the images look a little crusty and I'm worried LOL

**Review by LordofMankid — CHANGES_REQUESTED** [review]
lmao clicked wrong option make ur changes haha

File: src/app/(auth)/loginPage.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    <SafeAreaView
+      style={[
+        globalStyles.container,
+        { flex: 1, alignItems: "center", width: "100%" },
+      ]}
+    >
+      <KeyboardAvoidingView
+        style={{ width: "100%", flex: 1 }}
+        // keyboardVerticalOffset={200}
+        behavior={"position"}
```
Comment (johnny-t06) [inline src/app/(auth)/loginPage.tsx:35]: nit: remove any commented/unneeded code and imports up top! 
image
The grayed out lines are imports that are un-used. (Hopefully you can see it on your end as well, if not then I gotta figure out what vscode extension/theme does for me)

File: src/app/(auth)/signupPage.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    <SafeAreaView
+      style={[
+        globalStyles.container,
+        { flex: 1, alignItems: "center", width: "100%" },
+      ]}
+    >
+      <KeyboardAvoidingView
+        style={{ width: "100%", flex: 1 }}
+        // keyboardVerticalOffset={200}
+        behavior={"position"}
```
Comment (johnny-t06) [inline src/app/(auth)/signupPage.tsx:35]: nit: Remove commented lines if un-needed

File: src/components/login/SignupSection.tsx [hunk 4]
```diff
@@ -135,6 +146,20 @@ const SignupSection = () => {
             handlePassword(text);
           }}
         />
+
+        <View style={{ flex: 1, flexDirection: "row", gap: 16 }}>
+          <Image source={checkbox} style={{ width: 16, height: 16 }} />
+          <Text style={{ color: COLORS.neutral[70], fontSize: 11 }}>
```
Comment (johnny-t06) [inline src/components/login/SignupSection.tsx:155]: Checkbox here as well

**Review by johnny-t06 — CHANGES_REQUESTED** [review]
Great job team! The front end looks amazing. I left some comments about un-needed code that we could remove and agree with Clarence's comments! 

Feel free to re-request us when you address them!

**Pull request merged**